CHANGELOG.md

```diff
@@ -51,6 +51,11 @@ All notable changes to this project will be documented in this file.
     * #### Removed
 * ### `niscope` (NI-SCOPE)
     * #### Added
+        * Added methods for getting calibration information. - [#1463](https://github.com/ni/nimi-python/issues/1463)
```

Comment:
I think the word "Added" is redundant since this is a sub-bullet of the section "Added".